notebooks/recommender-system.ipynb

```diff
@@ -0,0 +1,843 @@
+{
```

Comment:
<p>Might help to describe "sparsity" here. </p><p> </p><p>There's a few steps now describing some of the data exploration steps (very important steps for reproducibility, but doesn't directly influence model). May help to add some headings:</p><ul><li>Data Exploration</li><li>Build model</li><li>train model</li><li>test model</li><li>wrap up</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cooperrc/uconn_numpy_tutorials/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>